qt/scientific_interfaces/Indirect/Reduction/ISISEnergyTransferModel.cpp

```diff
@@ -359,19 +359,26 @@ double IETModel::loadDetailedBalance(std::string const &filename) {
   return detailedBalance;
 }
 
-std::vector<std::string> IETModel::groupWorkspaces(std::string groupName, std::string groupOption) {
+std::vector<std::string> IETModel::groupWorkspaces(std::string groupName, std::string instrument,
+                                                   std::string groupOption, bool shouldGroup) {
```

Comment:
Most of these variables can be passed in as `const &`. This will mean we can avoid creating a copy of the variables when passing them into the function. The `bool shouldGroup` variable can be a `const` (with no `&`) because a `bool` is a primitive type and so we have no benefit passing it in by ref https://stackoverflow.com/questions/14013139/is-it-counter-productive-to-pass-primitive-types-by-reference

```suggestion
std::vector<std::string> IETModel::groupWorkspaces(std::string const &groupName, std::string const &instrument,
                                                   std::string const &groupOption, bool const shouldGroup) {
```